axelrod/tests/unit/test_thuemorse.py

```diff
@@ -0,0 +1,60 @@
+"""Test for the Thue-Morse strategies."""
+
+import axelrod
+from .test_player import TestPlayer
+
+C, D = axelrod.Actions.C, axelrod.Actions.D
+
+
+class TestThueMorse(TestPlayer):
+
+    name = 'ThueMorse'
+    player = axelrod.ThueMorse
+    expected_classifier = {
+        'memory_depth': float('Inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        """Test that strategy is randomly picked (not affected by history)."""
+        self.first_play_test(D)
+        self.first_play_test(D)
+
+    def test_effect_of_strategy(self):
+        self.markov_test(['C', 'C', 'C', 'C'])
+        self.responses_test([], [], [D, C, C, D, C, D, D, C, C, D, D, C, D, C, C, D])
+        self.responses_test([C], [C], [C, C, D, C, D, D, C, C, D, D, C, D, C, C, D])
+        self.responses_test([D], [D], [C, C, D, C, D, D, C, C, D, D, C, D, C, C, D])
+        self.responses_test([C, C, C, D], [C, C, C, D], [C, D, D, C, C, D, D, C, D, C, C, D])
+
+
+
+
+class TestThueMorseInverse(TestPlayer):
+
+    name = 'ThueMorseInverse'
+    player = axelrod.ThueMorseInverse
+    expected_classifier = {
+        'memory_depth': float('Inf'),
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def test_strategy(self):
+        """Test that strategy is randomly picked (not affected by history)."""
+        self.first_play_test(C)
+        self.first_play_test(C)
+
+    def test_effect_of_strategy(self):
+        self.markov_test(['D', 'D', 'D', 'D'])
+        self.responses_test([], [], [C, D, D, C, D, C, C, D, D, C, C, D, C, D, D, C])
+        self.responses_test([C], [C], [D, D, C, D, C, C, D, D, C, C, D, C, D, D, C])
```

Comment:
Could you perhaps as a docstring include the first couple of terms of the sequence just so that it's evident what the strategy is doing.